cli/tests/integrations/test_marathon_pod.py

```diff
@@ -220,7 +259,7 @@ def _pod(pod_id, file_path):
             _assert_pod_remove(pod_id, extra_args=['--force'])
 
 
-def _wait_for_instances(expected_instances, max_attempts=10):
+def _wait_for_instances(expected_instances, max_attempts=20):
```

Comment:
Sure, I don't even really need to test that scaling works at all. I'll just remove that from the update JSON.